setup.py

```diff
@@ -59,10 +59,18 @@ def get_gdal_config():
     gdal_version_str = os.environ.get("GDAL_VERSION")
 
     if include_dir and library_dir and gdal_version_str:
+        gdal_libs = ["gdal"]
+
+        if platform.system() == "Windows":
+            # NOTE: libgdal on conda-forge now builds the library as "gdal" instead
+            # of "gdal_i", but older Windows builds still use "gdal_i"
```

Comment:
The difference is not necessarily specific to conda-forge, but the it's whether `cmake` is used to build GDAL versus `nmake` (the old build system, which support will be removed in the next version of GDAL). For example I suppose that also our wheels using vcpkg will start using "gdal" if we update the version there.